packages/asset-uploader/src/delete.ts

```diff
@@ -0,0 +1,38 @@
+import { DeleteObjectCommand } from "@aws-sdk/client-s3";
+import { prisma, Asset } from "@webstudio-is/prisma-client";
+import { unlink } from "fs/promises";
+import path from "path";
+import { deleteAssetInDb } from "./db";
+import { imageFSDirectory } from "./helpers/image-fs-path";
+import { s3EnvVariables } from "./schema";
+import { getS3Client } from "./targets/s3/client";
+
+export const deleteAsset = async ({
+  id,
+  name,
+}: {
+  id: string;
+  name: string;
+}): Promise<Asset> => {
+  const currentAsset = await prisma?.asset.findUnique({
```

Comment:
how is prisma potentially not defined?
```suggestion
  const currentAsset = await prisma.asset.findUnique({
```

Reply:
done